src/nrniv/bgpdmasetup.cpp

```diff
@@ -598,8 +609,7 @@ static int setup_target_lists(int** r_return) {
 			}
 		}
 		delete tl;
```

Comment:
```suggestion
```
If we use `unique_ptr` then this is not needed, the `TarList` objects will be deallocated when `gid2tarlist` is destroyed.